processor/signozspanmetricsprocessor/processor.go

```diff
@@ -734,8 +733,8 @@ func (p *processorImp) buildDimensionKVs(serviceName string, span ptrace.Span, o
 	dims := pcommon.NewMap()
 	dims.PutStr(serviceNameKey, serviceName)
 	dims.PutStr(operationKey, span.Name())
-	dims.PutStr(spanKindKey, span.Kind().String())
-	dims.PutStr(statusCodeKey, span.Status().Code().String())
+	dims.PutStr(spanKindKey, "SPAN_KIND_"+strings.ToUpper(span.Kind().String()))
```

Comment:
Is this for backward compatibility?

Reply:
Correct. Otherwise, alerts/dashboards break.

Reply:
Cool.